toc.yml

```diff
@@ -600,6 +600,9 @@
       tocHref: /dotnet/machine-learning/
       topicHref: /dotnet/machine-learning/index
       items:
+      - name: How-to guides
+        tocHref: /dotnet/machine-learning/how-to-guides/
+        topicHref: /dotnet/machine-learning/how-to-guides/index.md
```

Comment:
this looks different than the others. It probably needs to be without md.

Reply:
ahhh ok